front-end/src/screens/Discussions/index.tsx

```diff
@@ -14,6 +15,17 @@ const Discussions = ({ className } : {className?: string}) => {
 		history.push('/post/create');
 	};
 
+	const infoBoxVisible = localStorage.getItem('discussionsInfo');
+	if (infoBoxVisible === null) {
+		localStorage.setItem('discussionsInfo', 'true');
```

Comment:
The name `discussionInfo` doen't ask a quesion that can be answered by true/false. It'd be preferable to keep the Visisble, hence `discussionInfoVisible`.